multisig/proposals/sepolia/vip-071/index.ts

```diff
@@ -0,0 +1,212 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { sepolia } = NETWORK_ADDRESSES;
+export const DEFAULT_PROXY_ADMIN = "0xe98a3110929c6650c73031756288Ec518f65e846";
```

Comment:
There are two Default Proxy Admin contracts on Sepolia :(

* 0xe98a3110929c6650c73031756288Ec518f65e846 (isolated pools)
* 0x01435866babd91311b1355cf3af488cca36db68e (rest of repos)

We should transfer the ownership of both contracts